kong/response.go

```diff
@@ -16,15 +17,26 @@ func newResponse(res *http.Response) *Response {
 	return &Response{Response: res}
 }
 
+func messageFromBody(b []byte) string {
+	s := struct {
+		Message string
+	}{}
+
+	if err := json.Unmarshal(b, &s); err != nil {
+		return fmt.Sprintf("<failed to parse response body: %v>", err)
```

Comment:
This `<>` is a new pattern in this codebase and I suggest to remove it (unless there is a reason for it that I don't know).

Reply:
The reason for `<>` is that this string is a "meta" message which takes place of the actual message, where obtaining the actual message failed.

Compare the following two messages printed to the user:
1. `HTTP status 400 (message: failed to parse response body)`
2. `HTTP status 400 (message: <failed to parse response body>)`

To me, 1 suggests that "failed to parse response body" is a message from the server, instead of conveying that the message from the server was unreadable or nonexistent.

No strong preference here though. The above being said, I'm happy to solve this problem in any other reasonable way.

Reply:
Is that a standard? This is the first time I'm hearing about this interpretation of `<>`.

I'm a little conflicted here. On one hand, I prefer consistency so I'm against it but on the other hand this seems too small to worry about consistency. I'm fine either ways and my approval reflects that.

Reply:
> Is that a standard?

Python does that when you try to print something unprintable.

I too think it's a minor detail - let's rethink it if a need arises in the future.